lib/fathead/nose2/parse.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+require 'nokogiri'
+require 'open-uri'
+
+NOSE_HOME = "http://nose2.readthedocs.io/en/latest/getting_started.html"
+topics = ARGV
+
+topics.each do |topic|
+  url = "http://nose2.readthedocs.io/en/latest/getting_started.html##{topic}"
```

Comment:
@rnkaufman this part of the documentation isn't ideal for a Fathead. Rather, I suggest you focus on http://nose2.readthedocs.io/en/latest/dev/event_reference.html and similar pages that define specific classes/functions/methods and other elements of the Nose2 API.

Those are each unique and specific, and when someone searches those we'll be able to show relevant documentation.

I rewrote a bunch of the Fathead documentation yesterday hoping to make this more clear.

Right now your Fathead only contains 4 articles, and the titles are very generic/vague/non-specific.

The article titles are the queries we expect to answer, so currently your fathead is going to show a result about Nose2 documentation when someone searches "dependencies" or "installation" -- clearly those results wouldn't be relevant to those queries.

You could still capture this content and prepend "nose2" to them so you'd have an article for "nose2 installation" -- that is way more specific!